src/site/navigation/sidebar_nav.drupal.liquid

```diff
@@ -1,30 +1,57 @@
-<nav id="va-detailpage-sidebar" class="usa-width-one-fourth va-sidebarnav">
+<nav id="va-detailpage-sidebar" class="va-drupal-sidebarnav usa-width-one-fourth va-sidebarnav">
     <div>
 
         <button class="va-btn-close-icon va-sidebarnav-close" type="button" aria-label="Close this menu"></button>
 
         {% for link in sidebar.links %}
             {% if forloop.first == true %}
                 <div class="left-side-nav-title">
-                    <i class="icon-small white hub-icon-{{ link.label }} hub-background-{{ link.label }}"></i>
+                    <i class="icon-small white hub-icon-{{ link.label | handleize }} hub-background-{{ link.label | handleize }}"></i>
                     <h4>{{ link.label }}</h4>
                 </div>
                 <ul class="usa-accordion">
                     {% for link in link.links %}
                         <li>
                             <button class="usa-accordion-button" aria-expanded="false" aria-controls="a{{ forloop.index }}">{{ link.label }}</button>
                             <div id="a{{ forloop.index }}" class="usa-accordion-content" aria-hidden="false">
```

Comment:
Should this menu also have the auto-expand behavior similar to `facility_sidebar_nav.drupal.liquid`?

Reply:
behavior is the same for both now.